domains/abdullah.json

```
@@ -5,7 +5,8 @@
     "username": "Mdabdullah3",
     "email": "mdabdullahnoman777@gmail.com"
   },
-  "record": {
-    "CNAME": "mdabdullah3.github.io"
+  "records": {
```

```suggestion
  "record": {
```

also you can't us CNAME with other records